src/main/java/frc/robot/pose/AlignmentSolution.java

```diff
@@ -0,0 +1,49 @@
+package frc.robot.pose;
+
+/**
+ * An alignment solution should include everything we need if we
+ * wanted to execute a fully autonomous score from our current position.
+ * 
+ * The AlignmentStrategy is how we'd pull it off, and the rest of the data
+ * should be the necessary details
+ * 
+ * Note: we also store a reference to the RobotPose we were given 
+ * as the basis for our calculations, and the scoring node we are going after
+ * @author dcowden
+ */
+public class AlignmentSolution {
```

Comment:
see dc_pose_updates branch: i think we need to move this onto RobotPose, so that we can get access to it everywhere.